packages/prisma-client/migrations-cli/commands.ts

```diff
@@ -318,3 +371,24 @@ export const resolve = async ({
   logger.info(`Resolved ${migrationName} as rolled back`);
   logger.info("");
 };
+
+export const reset = async () => {
+  // just to make it read the migrations folder
+  // and fail early if something is wrong with the migration files
+  await getStatus();
+
+  if (args.dev === false) {
```

Comment:
I think instead of coupling dev and reset, we could make reset coupled with either prompt for confirmation or something like --really-sure-want-this

Reply:
coupling with dev solves the problem of protecting prod database in theory, but still kinda implicit logic, also I could potentially connect my local environment to prod (I shouldn't) and then it would just kill the db still

Reply:
I'm trying to this similarly to Prisma https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-reset

<img width="649" alt="Screenshot 2022-09-07 at 17 23 11" src="https://user-images.githubusercontent.com/825702/188902820-e17634c5-d3fa-43cf-9e6e-0743bcf3f404.png">

Reply:
damn ... it makes sense to be compatible with them, --force flag is a good touch for this, because otherwise you still need to confirm and can't really do that by accident

Reply:
You can just run with `--dev` if you really want to do this

Reply:
I think you right actually! It feels kinda out of place to enforce this using `--dev`. And Prisma doesn't seem to do any enforcement.

Will allow to run without `--dev`, user will still have to either confirm or run with `--skip-confirmation`.